SQL/Cleanup_patches/2018-02-20_remove_duplicate_value_from_Config.sql

```diff
@@ -0,0 +1,10 @@
+/*  delete duplicate ConfigID-value pairs */
+
+Delete from Config  Where ID in
```

Comment:
No need for subqueries
`DELETE FROM Config USING Config, Config c1 WHERE Config.ID < c1.ID AND Config.ConfigID = c1.ConfigID AND Config.Value = c1.Value;`

Reply:
If you want to keep the olders then just change `Config.ID < c1.ID` for `Config.ID < c1.ID`

Reply:
@xlecours Thank you, you made it more clearly for everyone.